README.md

```diff
@@ -35,12 +35,20 @@ Here are some examples of how you could leverage `digitalocean-cloud-controller-
 * [loadbalancers](docs/controllers/services/examples/)
 * [node labels and addresses](docs/controllers/node/examples/)
 
-## Production advise
+## Production notes
+
+### do not modify DO load-balancers manually
 
 When creating load-balancers through CCM (via `LoadBalancer`-typed Services), it is important that you **must not change the DO load-balancer configuration manually.** Such changes will eventually be reverted by the reconciliation loop built into CCM. One exception are load-balancer names which can be changed (see also [the documentation on load-balancer ID annotations](/docs/getting-started.md#load-balancer-id-annotations)).
 
 Other than that, the only safe place to make load-balancer configuration changes is through the Service object.
 
+### DO load-balancer entry port restrictions
+
+For technical reasons, a few selected high ports cannot be used as load-balancer entry ports (i.e., the port that the load-balancer listens on for requests). Trying to use one of the affected ports as service port causes a _422 entry port is invalid_ HTTP error response to be returned by the DO API (and surfaced as a Kubernetes event).
```

Comment:
Can you mention the specific ports here too?

Reply:
Also would be great if we could link out to LB product docs for this when it's out.

Reply:
Ports are now called out.

Will submit a follow-up PR to point to the updated product docs as soon as those are available.